### ISSUE TITLE
File uploaded for the wrong task

### ISSUE THREAD

**Issue by Paulina-Rosetta** [TheRosettaFoundation/SOLAS-Match#1222]
A translator uploaded a file that he meant to upload for a different task. Can this task be reset to 'In Progress'? http://trommons.org/task/12601/view/

Many thanks

**Comment by alanbarrett** [comment]
I did: UPDATE Tasks SET `task-status_id`=3 where id=12601;
But I have now reversed that: UPDATE Tasks SET `task-status_id`=4 where id=12601;
Files are in: /repo/SOLAS-Match/backup/uploads/proj-3958
I don't know exactly what to delete without a good bit of investigation. A test of the simple approach on the dev server did not seem to work.
Also there is a TaskClaims table and possibly others.

So I think a work-around is better.

    Alan.
